samples/ai-image-tagging/src/index.js

```diff
@@ -85,14 +85,15 @@ class App extends React.Component {
   };
 
   render() {
-    const { contentType, parameters: { instance: { imageFieldId, tagFieldId } } } = this.props.sdk;
+    const { contentType, field, location, parameters: { instance: { imageFieldId, tagFieldId } } } = this.props.sdk;
     const { loadingTags, overwriteTags } = (this.state || {});
 
     const hasImageField = isCompatibleImageField(getField(contentType, imageFieldId));
     const hasTagField = isCompatibleTagField(getField(contentType, tagFieldId));
+    const isInCorrectLocation = location.is(locations.LOCATION_ENTRY_SIDEBAR);
 
     return (<div className='f36-color--text-light'>{
-      hasImageField && hasTagField ?
+      isInCorrectLocation && hasImageField && hasTagField ?
```

Comment:
This case logic seems to be complicated to me. I think it would be better to extract `isInCorrectLocation` from `ImageTaggingHelp` component and do something like this:

```
if (!location.is(locations.LOCATION_ENTRY_SIDEBAR)) {
   return <Note>The ai-image-tagging extension is an entry-level sidebar extension. You are currently using it on a field level.</Note>
}
return (
   <div className='f36-color--text-light'>
     {hasImageField && hasTagField ? <Component/> : < ImageTaggingHelp />}
   </div>
)


```

Reply:
Yeah makes sense, like using note for this as well.

Reply:
Extracted everything into their own components now, should be a lot easier to read